src/utils/traffic-mesh.js

```diff
@@ -92,6 +97,81 @@ const startForwardProxy = async ({ port, frameworkPort, functionsPort, publishDi
   }
 }
 
+const forwardMessagesToLog = ({ log, subprocess }) => {
+  let currentId = null
+  let spinner = null
+
+  const reset = () => {
+    currentId = null
+    spinner = null
+  }
+
+  rl.createInterface(subprocess.stderr)
```

Comment:
Should the `options` object syntax be used instead, since it is the [main one documented](https://nodejs.org/api/readline.html#readline_readline_createinterface_options)?

Reply:
Oh, I'm afraid I got things messed up there. I was actually referring to https://www.npmjs.com/package/readline when implementing this. Will take a look and ensure everything is correct!